dev_requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytest==4.1.1
+pytest==4.2.0
```

Comment:
Waiting for tests to pass.

I think we can modify this from `pytest==4.2.0` to `pytest`.

Reply:
if CI pass yes

Reply:
CI passed, I made the change.

Waiting for CI to re-pass :smile: